docs/bouwblokken/duurzaamheid/index.md

```diff
@@ -6,18 +6,32 @@ icon: material/leaf
 
 ![duurzaamheid](../../afbeeldingen/bouwblokken/duurzaamheid.jpg "visuele weergave duurzaamheid"){align="right"; width="50%"}
 
-Hier komt een beschrijving van dit bouwblok.
+Onze impact op natuur en milieu is groot. 
+Er zijn grote doelen gesteld om duurzamer te gaan leven en werken. Binnen alle overheidsorganisaties, op allerlei verschillende gebieden, wordt gekeken hoe er duurzamer te werk kan worden gegaan, dus ook bij ICT-voorzieningen. 
+
+Bij het duurzamer maken van ICT kan gedacht worden aan de fysieke kant (hardware) en de digitale kant (software, algoritmes).
+Met betrekking tot hardware kan men bijvoorbeeld zo duurzaam mogelijk hardware inkopen (circulariteit van apparaten en materialen) en proberen de levensduur van apparaten en onderdelen te maximaliseren.
```

Comment:
Nog geen soepel bruggetje naar de digitale (software matige kant)

Reply:
Dank voor je comments! Als het goed is, nu allemaal opgelost met mijn laatste commit.